pkg/systemd/quadlet/quadlet.go

```diff
@@ -38,6 +38,7 @@ const (
 	VolumeGroup     = "Volume"
 	ImageGroup      = "Image"
 	BuildGroup      = "Build"
+	QuadletGroup    = "Quadlet"
```

Comment:
You need to also define `XQuadletGroup    = "X-Quadlet"` and make sure to rename the group same as in https://github.com/containers/podman/blob/90fd1dce3420133bd6e472872ccfaa49dfe53f19/pkg/systemd/quadlet/quadlet.go#L542

Reply:
Done in every unit types